.github/workflows/ruby.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         ruby-version:
-          - jruby
+          - jruby-9.2
```

Comment:
This change is cherry-picked from https://github.com/mongomapper/mongomapper/pull/687/commits/6ba9ca82ba45f22861af6acd74737efaca53615c.